manifests/cf-manifest/cloud-config/020-cf-vm-types.yml

```diff
@@ -0,0 +1,125 @@
+---
+meta:
+  default_env:
+    bosh:
+      password: (( grab secrets.vcap_password ))
+
+vm_types:
+  - name: small
+    network: cf
+    env: (( grab meta.default_env ))
+    cloud_properties:
+      instance_type: m3.medium
```

Comment:
`m3.medium` is the smallest of the new instances. Normally T2 instances would be sufficient for this, but you only get 100 T2 instance starts per day per account (not sure if this is even per master account) which start with full CPU credits. As we auto-delete our environments, just the casual morning re-deployment of the environments exhausts this. When T2 instances start without CPU credits, they are _very_ slow.